tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -931,10 +932,35 @@ Ghost
         char_fx.color.a = alpha
         return true
 
+ .. code-tab:: csharp
+ 
+    using Godot;
+
+    [Tool]
+    public class RichTextGhost : RichTextEffect
+    {
+        // Syntax: [ghost freq=5.0 span=10.0][/ghost]
+
+        // Define the tag name.
+        private const string Bbcode = "ghost";
+
+        public override bool _ProcessCustomFx(CharFXTransform charFX)
+        {
+            // Get parameters, or use the provided default value if missing.
+            var speed = charFx.Env.Get("freq", 5.0f);
+            var span = charFx.Env.Get("span", 10.0f);
+
+            var alpha = Mathf.Sin(charFx.ElapsedTime * speed + (charFx.Range.X / span)) * 0.5f + 0.5f;
```

Comment:
Since the parameter name was changed.

```suggestion
            var alpha = Mathf.Sin(charFX.ElapsedTime * speed + (charFX.Range.X / span)) * 0.5f + 0.5f;
```